src/test_utils.jl

```diff
@@ -29,10 +29,11 @@ Test that a subtype of [`Descriptor`](@ref) implements the expected API.
 """
 function test_interface(D)
     @testset "test_interface $D" begin
-        @test FeatureDescriptors.sources(D) isa Vector{String}
+        # We don't care what sources returns, just that an applicable method is defined.
+        @test applicable(FeatureDescriptors.sources, D)
```

Comment:
tried this with `hasmethod` and it didn't work
```julia
julia> hasmethod(FeatureDescriptors.sources, Tuple{FakeDescriptor})
false
```

whereas `applicable` seems to do the trick
```
  applicable(f, args...) -> Bool

  Determine whether the given generic function has a method applicable to the given arguments.
```